### PR TITLE
Fixed fetching many-to-one relationships

### PR DESCRIPTION
**Pull request by connormeredith**
Fixes a bug where you couldn't fetch a photographer relationship on a photo resource.

### CODE REVIEW

File: lib/Resource.js [hunk 1]
```diff
@@ -267,9 +267,15 @@ Resource.prototype.fetch = Promise.denodeify(function(relationName, callback) {
...
-      return someResource._getBase();
-    });
+
+    if (!(newResources instanceof Array)) {
+      self._raw.relationships[relationName].data = newResources._getBase();
+    } else {
+      self._raw.relationships[relationName].data = newResources.map(function(someResource) {
+        return someResource._getBase();
+      });
+    }
```
Comment (pmcnr-hx) [inline lib/Resource.js:274]: Can we do this in a _return early_ like above, avoiding the `else` block, which should be more readable.

**Comment by reecemillsom** [comment]
taking 1️⃣

**Comment by reecemillsom** [comment]
👍

**Comment by pmcnr-hx** [comment]
Taking :two:

**Comment by pmcnr-hx** [comment]
:+1:

**Comment by theninj4** [comment]
:+1:

**Pull request merged**